js/components/main.js

```diff
@@ -1002,11 +997,9 @@ class Main extends ImmutableComponent {
             sortedFrames.map((frame) =>
               <Frame
                 ref={(node) => { this.frames[frame.get('key')] = node }}
-                tabData={this.props.appState.get('tabs').find((tab) => tab.get('tabId') === frame.get('tabId'))}
```

Comment:
@bridiver I don't remember the original use-case, but I did use this when implementing the per-tab alerts (but then removed in favor of a distinct `messageBoxDetail` property)